.prettierrc

```
@@ -0,0 +1,7 @@
+{
+  "useTabs": false,
+  "semi": true,
+  "singleQuote": true,
+  "trailingComma": "es5",
+  "bracketSpacing": true
```

I'd prefer only specifying options that are not the default (https://prettier.io/docs/en/options.html).
```suggestion
  "singleQuote": true
```

---

I might also prefer to set the `printWidth` to 100 to make the code a bit more condensed.